samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/ItemContentsBean.java

```diff
@@ -1717,5 +1718,13 @@ public Long getEffectiveItemId() {
 			return itemData.getItemId();
 		}
 	}
+
+	public boolean isCancelled() {
+		return this.itemData.getCancellation().intValue() != ItemDataIfc.ITEM_NOT_CANCELED;
```

Comment:
If using an integer, we can something like:

! ItemDataIfc.ITEM_NOT_CANCELED.equals(this.itemData.getCancellation())